data/movink.tablet

```diff
@@ -0,0 +1,45 @@
+# Wacom Movink DTH135K0C
+# Sensor Type: EMR
+# Features: Touch (Integrated), Tilt
+#
+# Button Map:
+# (A=1, B=2, C=3)
+#
+#    *------------------------------------------*
+#    |                    C                     |
+#  A |                                          | B
+#    |                                          |
+#    |                                          |
+#    |                                          |
+#    |                  TABLET                  |
+#    |                                          |
+#    |                                          |
+#    |                                          |
+#    |                                          |
+#    |                                          |
+#    *------------------------------------------*
+#
+# Note: touch data comes through the usb:056a:03f0 interface
+#
+# sysinfo.4XIn8KkXzp
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/376#issue-2270545906
+[Device]
+Name=Wacom Movink
+ModelName=DTH-135K0C
+DeviceMatch=usb|056a|03f0
+Class=Movink
```

Comment:
```suggestion
Class=Cintiq
```

`Class` is deprecated, no-one uses it and this is close enough to a cintiq so let's go with that.